ui/src/components/yaml/styles.module.less

```diff
@@ -359,261 +356,277 @@
+              margin-top: 24px;
+              margin-bottom: 16px;
+              font-weight: 600;
+              line-height: 1.25;
+            }
 
-          h2 {
-            font-size: 20px;
-          }
+            h1 {
+              font-size: 24px;
+            }
 
-          h3 {
-            font-size: 18px;
-          }
+            h2 {
+              font-size: 20px;
+            }
 
-          h4 {
-            font-size: 16px;
-          }
+            h3 {
+              font-size: 18px;
+            }
 
-          h5 {
-            font-size: 14px;
-          }
+            h4 {
+              font-size: 16px;
+            }
 
-          h6 {
-            font-size: 13px;
-          }
+            h5 {
+              font-size: 14px;
+            }
+
+            h6 {
+              font-size: 13px;
+            }
 
-          pre {
-            background: rgba(0, 0, 0, 0.4);
-            border: 1px solid rgba(147, 112, 219, 0.5);
-            border-radius: 6px;
-            padding: 12px;
-            margin: 12px 0;
-            overflow-x: auto;
+            pre {
+              background: rgba(0, 0, 0, 0.4);
+              border: 1px solid rgba(147, 112, 219, 0.5);
+              border-radius: 6px;
+              padding: 12px;
+              margin: 12px 0;
+              overflow-x: auto;
+
+              code {
+                background: transparent;
+                padding: 0;
+                color: #f0e6ff;
+              }
+            }
 
             code {
-              background: transparent;
-              padding: 0;
+              background: rgba(147, 112, 219, 0.2);
+              padding: 2px 6px;
+              border-radius: 4px;
               color: #f0e6ff;
+              font-family: 'Menlo', 'Monaco', 'Courier New', monospace;
             }
-          }
 
-          code {
-            background: rgba(147, 112, 219, 0.2);
-            padding: 2px 6px;
-            border-radius: 4px;
-            color: #f0e6ff;
-            font-family: 'Menlo', 'Monaco', 'Courier New', monospace;
-          }
-
-          ul,
-          ol {
-            padding-left: 24px;
-            margin: 8px 0;
+            ul,
+            ol {
+              padding-left: 24px;
+              margin: 8px 0;
 
-            li {
-              margin: 4px 0;
+              li {
+                margin: 4px 0;
+              }
             }
-          }
 
-          blockquote {
-            color: rgba(230, 230, 250, 0.85);
-            border-left: 4px solid rgba(147, 112, 219, 0.4);
-            background: rgba(147, 112, 219, 0.1);
-            margin: 16px 0;
-            padding: 12px 16px;
-            border-radius: 0 8px 8px 0;
-          }
+            blockquote {
+              color: rgba(230, 230, 250, 0.85);
+              border-left: 4px solid rgba(147, 112, 219, 0.4);
+              background: rgba(147, 112, 219, 0.1);
+              margin: 16px 0;
```

Comment:
Consider consolidating repeated styles for .dot and its nth-child animations to reduce redundancy.